packages/reports/tests/acceptance/report-test.js

```diff
@@ -103,6 +110,16 @@ module('Acceptance | Navi Report', function(hooks) {
     /* == Fix errors == */
     await clickItemFilter('dimension', 'Operating System');
     await clickItem('metric', 'Ad Clicks');
+    await clickItem('dimension', 'Date Time');
+    await clickItemFilter('dimension', 'Date Time');
+    await click($('.filter-builder__operator:contains(Since) .filter-builder__select-trigger')[0]);
+    await click($('li.ember-power-select-option:contains(Between)')[0]);
+
+    await clickTrigger('.filter-values--dimension-date-range-input__low-value .ember-basic-dropdown-trigger');
+
+    await click('.ember-power-calendar-day[data-date="2020-11-05"]');
```

Comment:
This will probably fail when we're no longer in november so maybe try to make these values dynamic

Reply:
Do we have any examples of this? would we just pick the first and last date of the calendar and how would we do that?

Reply:
We have this one where we pass in a moment and get an element selector for that date. So you could do something like `moment().startOf('month').add(4, 'days')` to `moment().startOf('month').add(5, 'days')`
https://github.com/yahoo/navi/blob/6a80ca7d51850f06b4d5a12b94538c1300f9a18f/packages/reports/tests/integration/components/navi-date-picker-test.js#L379-L383

or roughly the same version here

https://github.com/yahoo/navi/blob/9f2b5edede2134a93f16f307bdffaaee46ba5434/packages/reports/tests/acceptance/date-filter-test.js#L45-L50

Reply:
I chose the second version cause thats what I found initially in our code. Its pushed now